Sources/BrowserServicesKit/Autofill/AutofillUserScript.swift

```diff
@@ -45,28 +90,47 @@ public class AutofillUserScript: NSObject, UserScript {
         case pmHandlerOpenManagePasswords
     }
 
+    public var topAutofill: Bool
+    public var topView: OverlayProtocol?
+    public var clickPoint: NSPoint?
+    public var inputType: String?
+
     public weak var emailDelegate: AutofillEmailDelegate?
     public weak var vaultDelegate: AutofillSecureVaultDelegate?
     
     private var scriptSourceProvider: AutofillUserScriptSourceProvider
 
     public lazy var source: String = {
-        return scriptSourceProvider.source.replacingOccurrences(of: "PLACEHOLDER_SECRET", with: generatedSecret)
+        var js = scriptSourceProvider.source
+        js = js.replacingOccurrences(of: "PLACEHOLDER_SECRET", with: generatedSecret)
+        js = js.replacingOccurrences(of: "// INJECT isTopFrame HERE", with: "isTopFrame = \(topAutofill ? "true" : "false");")
```

Comment:
This is hacked back in, I don't really think there's much perf enhancements here especially over all the changes needed to land passing scriptSourceProvider to the top context.